gtsam/hybrid/HybridBayesNet.cpp

```diff
@@ -99,6 +99,7 @@ std::function<double(const Assignment<Key> &, double)> prunerFunc(
 }
 
 /* ************************************************************************* */
+// TODO(dellaert): what is this non-const method used for? Abolish it?
```

Comment:
This is used internally to update the BayesNet with the pruned conditionals after pruning them based on the `prunedDecisionTree`.